package/Dockerfile

```diff
@@ -160,9 +160,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
 # Versions for UI, DASHBOARD and CLI will be the same as 2.6 until new releases are made to these
```

Comment:
Does the comment need to be updated to ref 2.7?

Reply:
I dropped it because the info is not needed anymore.